### ISSUE TITLE
Make Cake wallet and Cupcake VoiceOver accessible

### ISSUE THREAD

**Issue by rottenwheel** [cake-tech/cake_wallet#3402]
**Is your feature request related to a problem? Please describe.**
A user asked in Matrix room whether developers could make both applications voiceover accessible. I know there's been some discussion and progress with these, but I figured an issue could come in handy, to track and discuss, precisely.

**Describe the solution you'd like**
At least, core features, such as balance, send, receive, transaction details should be voiceover accessible, for people who are blind, or have difficulty seeing things.

**Comment by rottenwheel** [comment]
cc. @shortwavesurfer2009.

**Comment by shortwavesurfer2009** [comment]
Cake wallet on Android is accessible, so cake wallet on iOS ought to be, or at least decently close, I would think.

I have not played with Cupcake on either platform.

**Comment by AnonymousAccessibilityAdvocate** [comment]
I have emailed Cakewallet about this issue and was told that it would be passed on to the developers. I hope this is addressed. Blind people should be able to enjoy the benefits of Cakewallet. With only minor tweaks needed to label elements for the screenreader, then I hope this issue is resolved quickly.
